.github/workflows/test.yml

```
@@ -0,0 +1,18 @@
+name: Multi threading Load Testing
+
+on:
+  push:
+    branch: testing/Testing-scripts
+
+jobs:
+  deploy:
```

Should this key really be called `deploy` or maybe the below or something else perhaps.
```suggestion
  test:
```